slack/users.py

```diff
@@ -30,3 +52,25 @@ def list():
     params = { 'token': slack.api_token }
 
     return slack.http_client.get('users.list', params)
+
+def setActive():
+    """
+    Sets the currently authenticated user as currently active
+    """
+    params = { 'token': slack.api_token }
+
+    return slack.http_client.get('users.setActive', params)
```

Comment:
should this be either POST or PUT?

It is not 100% clear from their doc. It would be great if you can check it by executing this.